Microsoft.Toolkit.Uwp/Helpers/Network/NetworkHelper.cs

```diff
@@ -0,0 +1,81 @@
+﻿// ******************************************************************
+// Copyright (c) Microsoft. All rights reserved.
+// This code is licensed under the MIT License (MIT).
+// THE CODE IS PROVIDED “AS IS”, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED,
+// INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+// IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+// DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
+// TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH
+// THE CODE OR THE USE OR OTHER DEALINGS IN THE CODE.
+// ******************************************************************
+
+using System;
+using Windows.Networking.Connectivity;
+
+namespace Microsoft.Toolkit.Uwp
+{
+    /// <summary>
+    /// This class exposes functionality of NetworkInformation through a singleton.
+    /// </summary>
+    public class NetworkHelper
+    {
+        private ConnectionInformation connectionInformation = null;
+
+        /// <summary>
+        /// Private singleton field.
+        /// </summary>
+        private static NetworkHelper _instance;
+
+        /// <summary>
+        /// Event raised when the network changes.
+        /// </summary>
+        public event EventHandler NetworkChanged;
+
+        /// <summary>
+        /// Gets public singleton property.
+        /// </summary>
+        public static NetworkHelper Instance => _instance ?? (_instance = new NetworkHelper());
+
+        /// <summary>
+        /// Gets instance of <see cref="ConnectionInformation"/>.
+        /// </summary>
+        public ConnectionInformation ConnectionInformation { get; private set; }
```

Comment:
Can be made readonly with GET only accessor? `public ConnectionInformation ConnectionInformation { get; }`